language/changes/6-phantom-type-params.md

```diff
@@ -0,0 +1,112 @@
+# Phantom Type Parameters
+
+- Status: Implemented in Move 1.4
+
+## Introduction
+
+A _phantom type parameter_ is one that doesn't show up at runtime, but is checked statically at
+compile time. Phantom type parameters can be useful in a couple of situations. For example, the
+struct `Event::EventHandle<T>` in the Move Standard library doesn't contain any field of type `T`,
+but uses the type parameter to ensure at the type level that the handle can only be used for
+messages of type `T`. Another prominent example is `Diem::Diem<CoinType>`, which is generic on a
+`CoinType` specifying the currency of the coin and allowing code to be written generically on any
+currency. As a last example, the capabilities in `Vault` also rely heavily on phantom type
+parameters.
+
+Previously, Move's type system didn't make a difference on whether a type parameter was phantom, but
+with this new feature, a struct's type parameter can be explicitly declared as phantom. A parameter
+declared as phantom is not considered when computing the
+[conditional abilities with generic types](3-abilities.md#Conditional-Abilities-with-Generic-Types).
+For this relaxed rule to be sound, Move's type system guarantees that a parameter declared as
+phantom is either not used at all in the struct definition, or it is only used as an argument to
+type parameters also declared as phantom.
+
+## Motivations
+
+Previously, defining a struct `S` with a phantom type parameter required spurious ability
+annotations to satisfy the requirements of the abilities declared for `S`. For a module author, this
+meant loosing control over structs that had to be declared with unnecessary abilities. Moreover, the
```

Comment:
```suggestion
meant losing control over structs that had to be declared with unnecessary abilities. Moreover, the
```

I'm not exactly sure what "losing control" means here. I was expecting something more along the lines of increased chance for bugs or security holes by weakening the type checking because of the unnecessary abilities.